<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestQTClass</class>
 <widget class="QMainWindow" name="TestQTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestQT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="priceListWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>320</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>326</x>
      <y>320</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Total price:</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>524</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="allTrenchCoatsLabel">
         <property name="text">
          <string>All trench coats:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="filterTrenchCoatsLineEdit">
         <property name="placeholderText">
          <string>Filter:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="allTrenchCoatsList"/>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="sizeLabel">
           <property name="text">
            <string>Size:</string>
           </property>
           <property name="buddy">
            <cstring>sizeLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="sizeLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="colourLabel">
           <property name="text">
            <string>Colour:</string>
           </property>
           <property name="buddy">
            <cstring>colourLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="colourLineEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="quantityLabel">
           <property name="text">
            <string>Quantity:</string>
           </property>
           <property name="buddy">
            <cstring>quantityLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="quantityLineEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="priceLabel">
           <property name="text">
            <string>Price:</string>
           </property>
           <property name="buddy">
            <cstring>priceLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="priceLineEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="photoLabel">
           <property name="text">
            <string>Photo:</string>
           </property>
           <property name="buddy">
            <cstring>photoLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="photoLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="addTrenchCoatButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeTrenchCoatButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="updateTrenchCoatButton">
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="shoppingBagLabel">
         <property name="text">
          <string>Shopping Bag:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="shoppingBagList"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="addToBagButton">
           <property name="text">
            <string>Add to Bag</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextButton">
           <property name="text">
            <string>Next</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buyButton">
           <property name="text">
            <string>Buy</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>589</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TestQT.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>populateBag()</slot>
  <slot>addToBag()</slot>
 </slots>
</ui>
